web/src/components/overview/OverviewPage.jsx

```diff
@@ -52,21 +49,24 @@ const IssuesList = ({ issues }) => {
   Object.entries(scopes).forEach(([scope, issues]) => {
     issues.forEach((issue, idx) => {
       const link = (
-        <ListItem key={idx}>
-          <Link to={`/${scope}`}>{issue.description}</Link>
-        </ListItem>
+        <Alert
+          key={idx}
+          isInline
+          variant={issue.severity === "error" ? "warning" : "info"}
+          title={<Link to={`/${scope}`}>{issue.description}</Link>}
+        />
       );
       list.push(link);
     });
   });
 
   return (
     <EmptyState
-      title={_("Before installing the system, you need to pay attention to the following tasks:")}
+      title={_("Before installing the system, you need to pay attention to the following tasks")}
```

Comment:
This text changed here. If you want to drop the colon (:), I would replace it with a stop (.).

Having said that, I do not know if the empty state plus the alert might be too much.

![Captura desde 2024-06-13 06-27-37](https://github.com/openSUSE/agama/assets/15836/f8918da8-79ca-4c41-8b25-d9ecea7cb442)

Reply:
Something went wrong from my side when merging the "adjust wording" commit here. Will check.

Reply:
> Having said that, I do not know if the empty state plus the alert might be too much.

Yes, it's too much. I was playing with a "NotificationDrawer" and also with some customs overrides, but I didn't found a convincing proposal yesterday. Let's try it again :)

Reply:
Fixed by picking up an idea I played yesterday and @joseivanlopez proposed this morning: using the PF/NotificationDrawer component. However, I've overwritten some styles to make it fit better in the overview page.

As disccussed offline, there is room for more improvements, but the current status should be enough for merging to master and continue from there.

| Problems | No problems |
|-|-|
|![Screenshot from 2024-06-13 10-30-50](https://github.com/openSUSE/agama/assets/1691872/d177d428-67f5-4c79-a993-68efcd6382f0) | ![Screenshot from 2024-06-13 10-31-02](https://github.com/openSUSE/agama/assets/1691872/89789d7f-d516-4549-881a-8c93581507e9) |